admin/includes/languages/english/lang.password_forgotten.php

```diff
@@ -17,7 +17,7 @@
     'ERROR_WRONG_EMAIL_NULL' => 'Go away gooberbrain :-P',
     'MESSAGE_PASSWORD_SENT' => 'Thank you. If the email address and username you entered matches an admin account in our database, then a new password will be sent to that email address.<br>Please read that email and then click "login" to use the new temporary password.',
     'TEXT_EMAIL_SUBJECT_PWD_RESET' => 'Your Requested change',
-    'TEXT_EMAIL_MESSAGE_PWD_RESET' => 'A new password was requested from %s.' . "\n\n" . 'Your new temporary password is:' . "\n\n   %s\n\nYou will be asked to choose a new password before logging in.\n\nThis temporary password expires in 24 hours.\n\n\n",
+    'TEXT_EMAIL_MESSAGE_PWD_RESET' => 'A new password was requested from %1$s.' . "\n\n" . 'Your new temporary password is:' . "\n\n   %2$s\n\nYou will be asked to choose a new password before logging in.\n\nThis temporary password expires in 24 hours.\n\n\n",
```

Comment:
Since the `%2$s` is within a double-quoted string, PHP will be looking for a `$s` variable to substitute. Suggest changing that last bit to
```php
"\n\n" . '%2$s' . "\n\nYou will be asked to choose a new password before logging in.\n\nThis temporary password expires in 24 hours.\n\n\n",
```

Reply:
Thanks!